### PR TITLE
Issues found by AB testing

### PR DESCRIPTION
**Pull request by natenichols**
* Websocket should be able to use `command` or `method
* when `ledger_index` is string, try to parse as a number

---
This change is [Reviewable](https://reviewable.io/reviews/cjcobb23/clio/110)

### CODE REVIEW

**Comment by natenichols** [comment]
Should have made this a draft PR, don't worry about reviewing

**Comment by natenichols** [comment]
Ok, so checks passed... I'll stop force pushing while this is being reviewed :)

File: src/rpc/RPCHelpers.cpp [hunk 1]
```diff
@@ -426,49 +503,154 @@ ledgerInfoToBlob(ripple::LedgerInfo const& info, bool includeHash)
...
+            {
+                cursor = AccountCursor({keys.back(), uNodeNext});
+                break;
+            }
+
+            if (uNodeNext == 0)
+                break;
+
+            currentIndex = ripple::keylet::page(rootIndex, uNodeNext);
+        }
```
Comment (cjcobb23) [inline src/rpc/RPCHelpers.cpp:618]: This loop is almost exactly the same as the loop in the below `else` body. Can we merge them?

File: src/webserver/WsBase.h [hunk 2]
```diff
@@ -243,6 +243,8 @@ class WsSession : public WsBase,
 
                 response = getDefaultWsResponse(id);
 
+                std::cout << response << std::endl;
+
```
Comment (cjcobb23) [inline src/webserver/WsBase.h:247]: Remove this cout

**Review by cjcobb23 — APPROVED** [review]

**Pull request merged**